resources/assets/styles/common/variables.scss

```diff
@@ -0,0 +1,19 @@
+$gutter: 8px;
+$gutter_num: 8;
+$transition: 0.4s;
+
+$colors: (
+  primary: #215f93,
+  secondary: #2c2a6d,
+  gray: #707070,
+  light-gray: #b3b2b5,
+  basic-white: #fff
```

Comment:
tutaj zmiana na basic-light lub light po prostu,  unikajmy definiowania nazwy koloru w zmiennej, jest to dopuszczalne w przypadku grey - bo odnosimy się po prostu do odcienia szarości, ale nie mamy pewności że nasz biały nie będzie w przyszłości np kremowym